.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2026-04-15 - Prevent SQL Injection in dynamically constructed native queries
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)